src/main/java/seedu/address/MainApp.java

```diff
@@ -74,6 +74,8 @@ public void init() throws Exception {
      * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
+        logger.info("Using data file : " + storage.getAddressBookFilePath());
```

Comment:
There looks like there is an extra space here before the colon

Current code:
```logger.info("Using data file : " + storage.getAddressBookFilePath());```

Proposed Change:
```logger.info("Using data file: " + storage.getAddressBookFilePath());```

Reply:
Hi sorry for the late update, the formatting is like to follow the other similar logging statements found in `MainApp` where a space is included before the colon.